src/test_workflow/integ_test/process_utils.py

```diff
@@ -0,0 +1,53 @@
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+import logging
+import os
+import subprocess
+
+import psutil  # type: ignore
+
+
+class ProcessUtils:
+
+    @staticmethod
+    def terminate_process(self):
```

Comment:
Since it's inside `Process`, just call it `terminate`.

Reply:
Sure. Addressed in https://github.com/opensearch-project/opensearch-build/pull/853/files